### PR TITLE
[rtmidi] init. was it because i made the pr from codespaces?

### PR DESCRIPTION
**Pull request by anandijain**

### CODE REVIEW

**Comment by anandijain** [comment]
evidently it was because of codespaces

**Comment by staticfloat** [comment]
Okay I built a new version of ALSA_jll; we'll wait for that to trickle through, then try again.

File: R/rtmidi/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "rtmidi"
+version = v"5.0.0"
```
Comment (wookay) [inline R/rtmidi/build_tarballs.jl:6]: I think that it should be `v"4.0.0"` .  https://github.com/thestk/rtmidi/blob/6e4e763/RtMidi.h#L61

**Review by wookay — CHANGES_REQUESTED** [review]

**Comment by anandijain** [comment]
what do I do about the platforms that failed?

**Pull request merged**